x-pack/plugins/security_solution/scripts/run_cypress/parallel.ts

```diff
@@ -70,8 +68,19 @@ ${JSON.stringify(argv, null, 2)}
       const cypressConfigFilePath = require.resolve(`../../${argv.configFile}`) as string;
       const cypressConfigFile = await import(cypressConfigFilePath);
 
-      if (cypressConfigFile.env?.TOOLING_LOG_LEVEL) {
-        createToolingLogger.defaultLogLevel = cypressConfigFile.env.TOOLING_LOG_LEVEL;
+      // Adjust tooling log level based on the `TOOLING_LOG_LEVEL` property, which can be
+      // defined in the cypress config file or set in the `env`
+      {
+        const logLevel =
+          process.env.TOOLING_LOG_LEVEL ||
+          process.env.CYPRESS_TOOLING_LOG_LEVEL ||
+          cypressConfigFile.env?.TOOLING_LOG_LEVEL ||
+          '';
+
+        if (logLevel) {
+          _cliLogger.info(`Setting tooling log level to [${logLevel}]`);
+          createToolingLogger.defaultLogLevel = logLevel;
```

Comment:
Is it expected behavior to affect all the instances created below in the execution flow?

As a safer option it might better to create a `ToolingLog` instance and pass it to `prefixedOutputLogger`.

Reply:
yeah... its expected. The goal is to ensure that all intances of `ToolingLog` (created by `createToolingLogger()`) uses by default the log level set via the env. variable or the cypress config file.